json_schema/python/README.md

```diff
@@ -0,0 +1,10 @@
+# JSON Schema validation with Python
+
+The code in this directory shows an example of using `jsonschema` for validation in combination with `pyyaml` YAML parser in Python.
+Usage:
+
+```bash
+$ python3 -m venv .venv
+$ source .venv/bin/activate
+$ pip install -r requirements.txt
+$ python validate.py
```

Comment:
```suggestion
```bash
python3 -m venv .venv
source .venv/bin/activate
pip install -r requirements.txt
python validate.py
```

PS. the last ` ``` ` were not commited 😢

Reply:
Yeah not sure what happened. I think github suggestion ate them :D I've fixed it.